src/hooks/useCurrentFireflyProfiles.ts

```diff
@@ -55,13 +55,16 @@ export function useCurrentFireflyProfilesAll() {
     const currentProfileAll = useCurrentProfileAll();
     const currentFireflyProfiles = useCurrentFireflyProfiles();
 
+    const lensIdentity = resolveProfileId(currentProfileAll[Source.Lens]);
+    const farcasterIdentity = resolveProfileId(currentProfileAll[Source.Farcaster]);
+
     const { data: profiles = EMPTY_LIST } = useQuery({
         queryKey: ['all-profiles', 'myself', currentProfileAll],
```

Comment:
```suggestion
        queryKey: ['all-profiles', 'my-own', lensIdentity, farcasterIdentity, twitterIdentity],
```

It's better to use critical values as `queryKey` instead of big object.